downloader/src/test/kotlin/TestTags.kt

```diff
@@ -0,0 +1,6 @@
+package com.here.ort.downloader
+
+import io.kotlintest.Tag
+
+object FunctionalTest : Tag()
+object IntegrationTest : Tag()
```

Comment:
I think we should only have one additional tag called e.g. `Expensive` (without `Test` in the end) and assign this tag to all tests that are resource expensive in some way, e.g. by taking up more CPU time than what we are willing to spend in PR checks. That said, not all tests you add in this PR necessarily need to have the tag assigned. It depends on how expensive they really are.